hector.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: index
+title: Climate Module – Hector	
+gcam-version: v4.2
+---
+
+This section describes the new climate module - Hector - that is available for use in GCAM. In GCAM, in addition to running MAGICC 5.3 (Wigley, 2008), there is the option to run Hector (Hartin et al., 2015).  Both Hector and MAGICC are reduced-form climate carbon-cycle models. 
+
+Hector, an open-source, object-oriented, reduced-form global climate carbon-cycle model, written in C++. This model runs essentially instantaneously while still representing the most critical global-scale earth system processes. Hector has a three-part main carbon cycle: a one-pool atmosphere, land, and ocean. The model’s terrestrial carbon cycle includes primary production and respiration fluxes, accommodating arbitrary geographic divisions into, e.g., ecological biomes or political units. Hector actively solves the inorganic carbon system in the surface ocean, directly calculating air– sea fluxes of carbon and ocean pH. Hector reproduces the global historical trends of atmospheric [CO<sub>2</sub>], radiative forcing, and surface temperatures. The model simulates all four Representative Concentration Pathways (RCPs) with equivalent rates of change of key variables over time compared to current observations, MAGICC (a well-known simple climate model), and models from the 5th Coupled Model Intercomparison Project. Hector’s flexibility, open-source nature, and modular design will facilitate a broad range of research in various areas. 
+
+![Hector Carbon Cycle diagram](gcam-figs/hector_box_model.png)<br/>
+Figure 1: Representation of Hector’s carbon cycle, land, atmosphere, and ocean. The atmosphere consists of one well-mixed box. The ocean consists of four boxes, with advection and water mass exchange simulating thermohaline circulation. At steady state, the high-latitude surface ocean takes up carbon from the atmosphere, while the low-latitude surface ocean off-gases carbon to the atmosphere. The land consists of a user-defined number of biomes or regions for vegetation, detritus and soil. At steady state the vegetation takes up carbon from the atmosphere while the detritus and soil release carbon back into the atmosphere. The earth pool is continually debited with each time step to act as a mass balance check on the carbon system. 
+
+
+For more information on Hector please see: 
```

Comment:
I don't really like this perfunctory "see-also" paragraph.  The two papers are in the References section, so we could just link them from there.  The GitHub link probably belongs in the section on getting and installing Hector.